docs/conf.py

```diff
@@ -53,8 +54,10 @@
 modindex_common_prefix = ["template_project."]
 
 # html theme options
-html_static_path = ["_static"]
+html_title = f"{project} {release}"
 html_logo = "_static/images/logo.png"
```

Comment:
This seems like a bad default. Few projects in the ecosystem have logos; it's more common to not have one. The Qiskit-sphinx-theme repo explains how to change the logo so you could consider cross-referencing there more. But I don't think the template should have the logo.

Reply:
Sure, I just commented it to let the people see it :)